workflows/iuf/operations/loftsman-manifest-deploy.yaml

```
@@ -107,6 +107,7 @@ spec:
                     else
                       echo "INFO Deploying loftman manifest ${manifest}"
                       manifest_deploy_output=$(loftsman ship --manifest-path /tmp/"$(basename $manifest)" --charts-repo https://packages.local/repository/charts 2>&1)
+                      echo "DEBUG $manifest_deploy_output"
```

```suggestion
                      echo >&2 "DEBUG $manifest_deploy_output"
```
Debug isn't "primary" output, so it could just be routed to stderr.